client/src/Components/Feed/Feed.js

```diff
@@ -36,60 +18,58 @@ const Feed = () => {
   const classes = useStyles();
 
   return (
-    <ThemeProvider theme={theme}>
-      <div className={classes.root}>
-        <List style={{ overflow: 'auto' }}>
-          <Grid container justifyContent="space-between" spacing={6}>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid item xs={4}>
-              <ItemCard />
-            </Grid>
-            <Grid
-              item
-              xs={12}
-              container
-              style={{ justifyContent: 'center', marginBottom: '30px' }}
-            >
-              <Button variant="contained" spacing={1} color="secondary">
-                Show More
-              </Button>
-            </Grid>
+    <div className={classes.root}>
+      <List style={{ overflow: 'auto' }}>
```

Comment:
Is the list necessary here? I think the grid will automatically overflow into the next row.